src/FortniteSettingsParser/Object/FortniteSettingsHeader.cs

```diff
@@ -15,9 +30,31 @@ public class FortniteSettingsHeader
 
         public FortniteSettingsHeader(UnrealBinaryReader reader)
         {
-            Unknown1 = reader.ReadBytes(18);
+            Something = reader.ReadInt16();
+
+            reader.Seek(2, SeekOrigin.Current);
+
+            FileVersionUE4 = reader.ReadInt32();
+
+            if (Something != 2464) {
```

Comment:
i dont really know if this makes sense but it seems to work with all versions i tested